docs/config-file-curations-yml.md

```diff
@@ -73,6 +73,8 @@ The structure of the curations file consist of one or more `id` entries:
     is_meta_data_only: true
```

Comment:
Commit message nit: The last part of the sentence seems to be incomplete. Should it be
"...curation, so *they* can learn *about* available options." ?

Reply:
Good catch, fixed